Features/Context/Users.php

```diff
@@ -36,6 +36,17 @@ public function iCreateUser(TableNode $users = null)
     }
 
     /**
+     * @When I go to (the) User :username page
+     */
+    public function goToUserPage($username)
+    {
+        $this->iAmOnPage('Users');
+        $this->waitWhileLoading();
+        $this->clickOnTreePath("$username $username");
```

Comment:
This is a context problem, connected to what I have said above.

We should/could store the created user's reference, so that we can easily navigate to it. It would avoid having to reference the same fake string, and we could use `uniqid()` in generated names to ensure uniqueness.